Agents/EnergyPerformanceCertificateAgent/agent/kgutils/querytemplates.py

```diff
@@ -157,10 +157,13 @@ def get_children_and_parent_building_properties():
             OPTIONAL {{ ?address_iri <{OBE_HAS_UNIT_NAME}> ?addr_unit_name }}
             OPTIONAL {{ ?property_iri <{OBE_HAS_ENERGYRATING}> ?epc_rating }}
             OPTIONAL {{ ?property_iri <{OBE_HAS_NUMBER_ROOMS}> ?rooms }}
-            OPTIONAL {{ ?property_iri <{OBE_HAS_USAGE}> ?usage_iri }}
+            OPTIONAL {{ ?property_iri <{OBE_HAS_PROPERTY_USAGE}> ?usage .
+                        ?usage <{RDF_TYPE}> ?usage_iri }}
             OPTIONAL {{ ?property_iri <{RDFS_LABEL}> ?usage_label }}
```

Comment:
Removed the line "OPTIONAL {{ ?property_iri <{RDFS_LABEL}> ?usage_label }}" from this method, and another method, where the parent building data is queried, because this relationship is invalid now (Usage labels are attached to usage instances rather than property instances).
As discussed earlier, did not switch the `usage` and `usage_iri` variables.